### PR TITLE
update header, footer, remove css, point to component library

### LINKED ISSUE #518: Layout needs to be updated to follow modern brand guidelines
This is currently using a bunch of beige colors, but we need to follow the brand identity guide: https://identity.stanford.edu/design-elements/color/web/  Purl is a good example of a site that has been converted already.

### PR DESCRIPTION
**Pull request by dnoneill**
closes #518 

image

### CODE REVIEW

**Comment by jcoyne** [comment]
@dnoneill is the black header requested by a PO or UX person?

**Comment by jcoyne** [comment]
I believe we should also remove the `#su-wrap` and `#su-content` containers here.

**Comment by dnoneill** [comment]
@jcoyne Nope. Should I go back to the cardinal-red? I was under the impression we were moving away from that but I am happy to put it in.

File: app/assets/stylesheets/layout.scss [hunk 1]
```diff
@@ -1,43 +1,29 @@
...
     min-height: 0;
   }
 }
 
-#sul-logo {
-  border-bottom: 0;
-  float: left;
-}
-
 @media (min-width: 768px) {
```
[the remark this answers is not in the document]
Comment (dnoneill) [reply app/assets/stylesheets/layout.scss]: done

**Comment by dnoneill** [comment]
removed those containers as well.

**Comment by jcoyne** [comment]
@dnoneill yes, I'd suggest `.bg-cardinal` on those unless otherwise specified.

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -29,18 +30,14 @@
...
-          <%= render 'shared/top_navbar' %>
-          <section id="main-container" role="main" class="container pt-2 pb-3">
-            <%= breadcrumbs style: :bootstrap5 %>
-            <%= render 'shared/global_alert' %>
-            <%= render 'shared/flashes' %>
-            <%= yield %>
-          </section>
-        </div>
-      </div>
+    <div id="outer-container" class="container-fluid">
```
Comment (jcoyne) [inline app/views/layouts/application.html.erb:33]: Sorry, I wonder if this one is needed too? In other sites we don't have this.

File: app/views/layouts/application.html.erb [hunk 3]
```diff
@@ -29,18 +30,14 @@
...
+      <%= render 'shared/top_navbar' %>
+      <section id="main-container" role="main" class="container pt-2 pb-3">
```
Comment (jcoyne) [inline app/views/layouts/application.html.erb:35]:
```suggestion
      <main id="main-container" class="container pt-2 pb-3">
```

File: app/views/layouts/application.html.erb [hunk 4]
```diff
@@ -29,18 +30,14 @@
...
+        <%= breadcrumbs style: :bootstrap5 %>
+        <%= render 'shared/global_alert' %>
+        <%= render 'shared/flashes' %>
+        <%= yield %>
+      </section>
```
Comment (jcoyne) [inline app/views/layouts/application.html.erb:40]:
```suggestion
      </main>
```

File: app/views/layouts/application.html.erb (hunk 2 above)
Comment (dnoneill) [reply app/views/layouts/application.html.erb:33]: We are going to need some class I think. I tried to pull out https://github.com/sul-dlss/library_hours_rails/pull/585/files#diff-4b04ba88a773a7736d1e8a5bcb978cf7faa74c935b4389bcd9d962407c9a8149R1-R5

Comment (dnoneill) [reply app/views/layouts/application.html.erb:33]: Actually removing it totally fixes this. Getting rid of container fluid.

**Comment by jcoyne** [comment]
It looks like the link color changed from brownish to blue.  Is this intentional?

**Comment by jcoyne** [comment]
Alan said on Slack:
> Looking at the redesigned Requests app as an example, I think: the blue links are intentional; date range doesn't need a background color; Go button should be blue.

**Comment by corylown** [comment]
@dnoneill looks like the app login link got removed in this PR: https://github.com/sul-dlss/library_hours_rails/pull/585/files#diff-380fa1d5663a0cfe9174a52172faeed9de558db0e2b60267d672f9df9993c8d2L15

**Pull request merged**